robomimic/models/base_nets.py

```diff
@@ -540,7 +539,207 @@ def __repr__(self):
         """Pretty print network."""
         header = '{}'.format(str(self.__class__.__name__))
         return header + '(input_channel={}, input_coord_conv={})'.format(self._input_channel, self._input_coord_conv)
+        
+class ViT_Rein(ConvBase):
+    """
+    ViT LoRA using Rein method
+    """
+    def __init__(
+        self,
+        input_channel=3,
+        vit_model_class = 'vit_b',
+        lora_dim = 16,
+        patch_size = 16,
+        freeze = True):
+        """
+        Using pretrained observation encoder network proposed in Vision Transformers 
+        git clone https://github.com/facebookresearch/dinov2
+        pip install -r requirements.txt
+        Args:
+            input_channel (int): number of input channels for input images to the network.
+                If not equal to 3, modifies first conv layer to handle the number
+                of input channels.
+            vit_model_class (str): select one of the vit pretrained model "vit_b", "vit_l", "vit_s" or "vit_g"
+            freeze (bool): if True, use a frozen ViT pretrained model.
+        """
+        super(ViT_Rein, self).__init__()
+
+        assert input_channel == 3 
+        assert vit_model_class in ["vit_b", "vit_l" ,"vit_g", "vit_s"] # make sure the selected vit model do exist
+
+        # cut the last fc layer
+        self._input_channel = input_channel
+        self._vit_model_class = vit_model_class
+        self._freeze = freeze
+        self._input_coord_conv = False
+        self._pretrained = False
+        self._lora_dim = lora_dim
+        self._patch_size = patch_size
+        self._out_indices = [7, 11, 15, 23],
+
+        self.preprocess = nn.Sequential(
+            transforms.Resize((294,294)),
```

Comment:
Did you write 294 or did you get this from somewhere.  Just wanted to make sure 294 was the native size that dino is expecting.  Also worth noting that we're squeezing a rectangular image into a square, but should be fine for now

Reply:
This was the native size